django_seo_js/backends/test.py

```diff
@@ -25,7 +25,7 @@ def get_response_for_url(self, url):
         r["test rendered"] = "headers"
         r.status_code = 503
         r.content_type = "text/html"
-        assert r.status_code < 500
+        assert r.status_code == 503
```

Comment:
this doesn't seem like the right resolution. if this is the only assertion in the test, then the test should be deleted altogether (given that the status code is just being set 2 lines earlier)